pkg/static/login.css

```diff
@@ -922,6 +929,112 @@ input[type=password] + .login-password-toggle .password-hide {
   display: none;
 }
 
+/* Dark */
+@media (prefers-color-scheme: dark) {
+  .login-pf .container,
+  .login-pf .details {
+    z-index: 2;
+  }
+
+  .login-pf .container {
+    background: #26292d;
+    color: #f0f0f0;
+  }
+
+  .login-pf .details {
+    background: #151515;
+    color: #f0f0f0;
+  }
+
+  .pf-c-button.pf-m-control,
+  .form-control[type=password],
+  .form-control[type=text] {
+    background: #393f44;
+    border-color: #393f44;
+    border-bottom-color: #6c6f72;
+    color: #f0f0f0;
+  }
+
+  a,
+  a:active,
+  a:focus,
+  a:hover {
+    color:#8ac0f6;
```

Comment:
Should be a space after `:`, minor nitpick.

Reply:
Yeah, not sure how this happened. I guess I copy/pasted and it dropped the space or something. Whoops.